lib/Dialect/AIEVec/Transforms/AIEVectorize.cpp

```diff
@@ -40,6 +40,9 @@ static llvm::cl::opt<bool>
                         llvm::cl::desc("Enable the unaligned loads check"),
                         llvm::cl::init(true));
 
+static llvm::cl::opt<bool> AIEML("aieml", llvm::cl::desc("AI Engine-ML"),
```

Comment:
> In the future we may need to consider more variants. I suggest an enum-style command line option "aiearch" for with (currently) 2 legal values "aie" and "aie-ml". This will make it more consistent with other tool chains. The default value would be "aie"

Agree.